docs/index-shared.asciidoc

```diff
@@ -0,0 +1,64 @@
+
+//////////
+release-state can be: released | prerelease | unreleased
+//////////
+:release-state:   unreleased
+:version:         5.5.0
+:major-version:   5.x
+:branch:          5.x
+
+:docker-image:    docker.elastic.co/kibana/kibana:{version}
+:es-ref:          https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
+:xpack-ref:       https://www.elastic.co/guide/en/x-pack/current/
+:repo:            https://github.com/elastic/kibana/
+:issue:           {repo}issues/
+:pull:            {repo}pull/
+:commit:          {repo}commit/
+:blob:            {repo}blob/{branch}/
+:security:        https://www.elastic.co/community/security/
+:xpack:           X-Pack
+:security:        X-Pack security
+:monitoring:      X-Pack monitoring
+:watcher:         Watcher
+:reporting:       X-Pack reporting
+:graph:           X-Pack graph
+:searchprofiler:  X-Pack search profiler
+:xpackml:         X-Pack machine learning
+:es:              Elasticsearch
+:kib:             Kibana
+
+ifeval::["{include-xpack}"=="true"]
+:xkb-repo-dir: {docdir}/../../x-pack-kibana/docs/en
+endif::[]
+
+include::introduction.asciidoc[]
+
+include::setup.asciidoc[]
+
+include::migration.asciidoc[]
+
+include::getting-started.asciidoc[]
+
+include::discover.asciidoc[]
+
+include::visualize.asciidoc[]
+
+include::dashboard.asciidoc[]
+
+include::timelion.asciidoc[]
+
+include::console.asciidoc[]
+
+include::management.asciidoc[]
+
+ifeval::["{include-xpack}"=="true"]
```

Comment:
Sorry, as you point out this syntax doesn't work for includes:

```
{include-xpack#}include::....
```

However this syntax does:

```
ifdef::include-xpack[]
include::{xkb-repo-dir}/reporting/index.asciidoc[]
endif::include-xpack[]
```

and it slightly lighter than `ifeval`